reader-osm/src/test/java/com/graphhopper/routing/RoutingAlgorithmWithOSMIT.java

```diff
@@ -509,6 +522,8 @@ Graph runAlgo(TestAlgoCollector testCollector, String osmFile,
                     setGraphHopperLocation(graphFile).
                     setEncodingManager(new EncodingManager(importVehicles));
 
+            if (osmFile.contains("krautsand"))
+                hopper.setMinNetworkSize(0, 0);
```

Comment:
Not really beautiful but I didn't wanted to introduce a new parameter or method just because of this